checker/src/path.rs

```diff
@@ -65,7 +54,7 @@ impl From<Rc<Path>> for PathWithHash {
 /// When the actual expression is evaluated at runtime it will resolve to a particular memory
 /// location. During analysis it is used to keep track of state changes.
 #[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
-pub enum Path {
+pub enum PathEnum {
```

Comment:
nit: Kinda meh about the name PathEnum. PathVariant, PathKind? PathAccess? I guess it's all meh :)

Reply:
Call an enum an enum.